lib/VMwareWebService/broker_timeout.rb

```diff
@@ -0,0 +1,64 @@
+module Timeout
```

Comment:
Great work everyone.  Really looking forward to fixing this upstream in some form :+1: